docs/proposals/2019-04-23 - Audio.md

```diff
@@ -0,0 +1,50 @@
+# Problem
+
+Currently, audio objects are not in the Content API.  We are now at a point where we would like to add audio objects into the Content API and are honing the audio object to include all of the necessary elements into the json.  Thus, I propose that embed_html, series, and audio_type be added to the json.
+
+
+# Proposal
+
+Add embed_html, series, and audio_type to audio object.
+
+
+## Embed_html
+
+HTML for the embedded audio player
+
+## Series
```

Comment:
How might this be used for non-podcast types? Does this name make sense for other types of audio?

Reply:
Yeah I was thinking something like `category` might make more sense for that same reason ^